src/components/Users/Users.js

```diff
@@ -1,12 +1,30 @@
-import User from '../User/User';
-import './Users.css';
+import User from "../User/User";
+import "./Users.css";
+
+const Users = ({ users, setExpanded, expanded }) => {  
+  const handleExpanded = (id) => {
+    let copyExpaned = expanded;
```

Comment:
I think it's unnecessary to use `copyExpanded` here, for a couple of reasons.

1. `copyExpanded = expanded` doesn't actually copy it. Both `copyExpanded` and `expanded` point to the same array; no copy is made.
2. In both the `if` and `else` blocks, you use safe non-mutating methods that return a new array (`filter` and `[...copyExpanded, id]`)

Even if `copyExpanded` was a true copy, this would be overkill because in 2) we see that you are already safely handling the original array.